src/main/webapp/assets/js/index.js

```diff
@@ -1,26 +1,26 @@
 // default to load 6 product first
-const INITIAL_PRODUCT_LOAD = 6
-const INITIAL_DESC_WORDS = 20
-
-async function getMultipleMockData() {
-  let response = await fetch('/multiplemockdatabusiness');
-  let mockdata = await response.json();
-  return mockdata;
-}
+const INITIAL_PRODUCT_LOAD = 6;
+const INITIAL_DESC_WORDS = 20;
 
 /**
  * Initialize the home page with product listings.
  */
 async function initHomePage() {
-  const mockData = await getMultipleMockData();
-  const productsToLoad = Math.min(INITIAL_PRODUCT_LOAD, mockData.length);
-  let foodLocations = []
+  const business = await getBusinessData();
+  const productsToLoad = Math.min(INITIAL_PRODUCT_LOAD, business.length);
+  let foodLocations = [];
 
   for (let i = 0; i < productsToLoad; i++) {
-    document.getElementById("product-listings").innerHTML += homePageListingTemplate(mockData[i]);
-    foodLocations.push(createLocation(mockData[i]))
+    document.getElementById("product-listings").innerHTML += homePageListingTemplate(business[i]);
```

Comment:
consider using appendChild instead of concatenating innerHTML += (the latter may be considered "bad code")

https://stackoverflow.com/questions/11515383/why-is-element-innerhtml-bad-code

Reply:
will fix! thanks